### PR TITLE
step 1 90%

### PR DESCRIPTION
**Pull request by Karakrisz**
**PLEASE NOTE:**

This is only a issue tracker for issues related to the CakePHP Application Skeleton.
For CakePHP Framework issues please use this [issue tracker](https://github.com/cakephp/cakephp/issues).

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

The best way to propose a feature is to open an issue first and discuss your ideas there before implementing them.

Always follow the [contribution guidelines](https://github.com/cakephp/cakephp/blob/master/.github/CONTRIBUTING.md) guidelines when submitting a pull request. In particular, make sure existing tests still pass, and add tests for all new behavior. When fixing a bug, you may want to add a test to verify the fix.

### CODE REVIEW

File: src/Template/Visits/add.ctp [hunk 1]
```diff
@@ -48,12 +50,16 @@
                                 echo $this->Form->control('club.name',['label' => __('Club')]);
                             } */
                             ?>
+                            <div class="input-group">
+                                <i class="fi-calendar"> </i>
                             <?= $this->Form->control('date',
                                 ['id' => 'datepicker', 'type' => 'text',
                                     'label' => __('Date of visit'),
+                                    'placeholder' => __('Látogatás dátuma') ,
```
Comment (rrd108) [inline src/Template/Visits/add.ctp]: Tedd vissza az angol szöveget

File: src/Template/Visits/add.ctp [hunk 2]
```diff
@@ -64,7 +70,7 @@
                                             'class' => 'column large-8',
                                             'escape' => false
                                         ],
-                                    'placeholder' => __('Full price') . ' (' . __('person') . ')',
+                                    'placeholder' => __('Felnőtt') . ' (' . __('fő') . ')',
```
Comment (rrd108) [inline src/Template/Visits/add.ctp]: Szintén vissza az angolt

File: src/Template/Visits/add.ctp [hunk 3]
```diff
@@ -74,6 +80,7 @@
                             ) ?>
                             </div>
                             <div class="input-group">
+                                <i class="fi-universal-access" style="font-size: 50px; position: relative; bottom:20px; right:5px;"> </i>
```
Comment (rrd108) [inline src/Template/Visits/add.ctp]: ne használj inline css-t, tedd át a css-be

File: src/Template/Visits/add.ctp [hunk 4]
```diff
@@ -84,7 +91,7 @@
                                             'class' => 'column large-8',
                                             'escape' => false
                                         ],
-                                    'placeholder' => __('Discount price') . ' (' . __('person') . ')',
+                                    'placeholder' => __('Diák, Nyugdíjas') . ' (' . __('fő') . ')',
```
Comment (rrd108) [inline src/Template/Visits/add.ctp]: angolt vissza

File: webroot/css/visitors.css [hunk 5]
```diff
@@ -4,8 +4,54 @@
...
-}
-
+    background: #DECF9B;
+}
+nav img {
+    width: 200px;
+    height: auto;
+
+}
+@media screen and (max-width: 675px) {
```
Comment (rrd108) [inline webroot/css/visitors.css:14]: szerintem nem kell media query. Most egcsináljuk  a mobil megjelenést, és ha az kész annak a tetejére tennék media query-vel a desktopot

**Comment by Karakrisz** [comment]
első szakasz

**Review by rrd108 — CHANGES_REQUESTED** [review]

**Comment by Karakrisz** [comment]
javitva

**Review by rrd108 — APPROVED** [review]

**Pull request merged**